src/server/sass/build.js

```diff
@@ -75,11 +78,45 @@ export class Build {
       importer: this.theme === 'dark' ? DARK_THEME_IMPORTER : undefined
     });
 
-    const prefixed = postcss([ autoprefixer ]).process(rendered.css);
-
     this.includedFiles = rendered.stats.includedFiles;
 
-    await mkdirpAsync(path.dirname(this.targetPath));
+    const sourceBaseDir = path.dirname(this.sourcePath);
+    const targetDir = path.dirname(this.targetPath);
+    const processor = postcss([ autoprefixer ]);
+
+    if (this.urlImports) {
+      const { publicDir, urlBase } = this.urlImports;
+      processor.use(postcssUrl({
+        url: (asset) => {
+          if (!asset.pathname) {
+            return asset.url;
+          }
+
+          const sourcePath = path.resolve(sourceBaseDir, asset.pathname);
+          if (!fs.existsSync(sourcePath)) {
+            throw new Error(`Unable to locate url("${asset.url}"), it must resolve to a file relative to "${publicDir}"`);
```

Comment:
_publicDir_ should be _sourceBaseDir_ or _this.sourcePath_

```suggestion
            throw new Error(`Unable to locate url("${asset.url}"), it must resolve to a file relative to "${sourceBaseDir}"`);
```